content/en/blog/2023-03-01-february-update/index.md

```diff
@@ -0,0 +1,259 @@
+happened in March - it has been a lot!
+
+## News in the Flux family
+
+### Next Flux release: more stability and performance improvements
+
+We have released [Flux v0.39](https://github.com/fluxcd/flux2/releases/tag/v0.39.0).
+Users are encouraged to upgrade for the best experience.
+
+Highlights
+
+- Starting with this version, the Flux controllers come with [SBOMs and
+  SLSA Provenance Attestations](/flux/security/)
+  embedded in their container images.
+- The [Flux Terraform Provider](https://github.com/fluxcd/terraform-provider-flux)
+  has a new resource for bootstrapping Flux, without depending on
+  third-party Terraform providers, that allows customising the
+  controllers at install time. Users are encouraged to migrate to
+  this new resource and provide feedback.
+- The Flux CLI is now included in [Wolfi OS](https://github.com/wolfi-dev/os),
+  the Linux (Un)distro designed for securing the software supply chain. The
+  Chainguard team and Wolfi maintainers are shipping updates for the Flux
+  package on a regular basis.
+
+Features and improvements
+
+- Recreate immutable resources (e.g. Kubernetes Jobs) by annotating
+  or labeling them with `kustomize.toolkit.fluxcd.io/force: enabled`.
+- Support for HTTPS bearer token authentication for Git repositories.
+- Improve memory usage by disabling the caching of `Secret` and
+  `ConfigMap` resources in all controllers.
+- Better observability with progressive status updates for Sources
+  (Git, OCI, Helm, S3 Buckets).
+- Allow extracting the OCI artifact SHA256 digest for Cosign with
+  `flux push artifact -o json`.
+- Track CRDs managed by Flux, `flux trace` and `flux tree` will show
+  which HelmRelease deployed which CRDs.
+- Allow the Flux GitHub Action to use a GitHub token when checking
+  for updates to avoid rate limiting.
+
+Documentation
+
+- Security: [Software Bill of Materials](/flux/security/#software-bill-of-materials)
+- Security: [SLSA Provenance Attestations](/flux/security/#slsa-provenance-attestations)
+- Security: [Scanning Flux images for CVEs](/flux/security/#scanning-for-cves)
+
+Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.17.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.16.0>
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
```

Comment:
@juozasg What shall we put in here for February?

Reply:
I'm aiming for a release of this update on Wednesday.

Reply:
Version 0.23.0 of vscode-gitops-tools extension was released. This version introduces a new webview for for configuring GitRepository, HelmRepository, OCIRepository, Bucket and Kustomization resources. Extension context (right-click) actions now work with multiple open repositories in the expected way.  

![](https://github.com/weaveworks/vscode-gitops-tools/blob/main/docs/images/vscode-configure-gitops.png)